--- gofish/Configure_GoFish.orig	2010-10-04 05:03:49.000000000 +0200
+++ gofish/Configure_GoFish	2026-04-11 16:06:45.115475267 +0200
@@ -7,7 +7,7 @@ which where provided as part of the GoFi
 
 For the really impatient:
 
-    1) Add some files and/or directories to /var/gopher
+    1) Add some files and/or directories to /home/services/gopherd
     2) Run `mkcache -r'
     3) Sit back and wait for the hits
 
@@ -19,7 +19,7 @@ The Gopher spec is defined in RFC 1436 "
 Links below).  The important thing to remember is that gopher is ment
 to share files. It uses a directory structure to layout the files. The
 root directory is the base directory that all files descend from. In
-GoFish the root directory defaults to /var/gopher.
+GoFish the root directory defaults to /home/services/gopherd.
 
 GoFish requires a special file called ".cache" in each directory you
 want to serve files from. There is a man page provided with GoFish on
--- gofish/check-files.orig	2010-10-04 05:03:49.000000000 +0200
+++ gofish/check-files	2026-04-11 16:09:58.377761609 +0200
@@ -1,6 +1,6 @@
 #!/bin/sh
 
-cd /var/gopher
+cd /home/services/gopherd
 
 for cache in `find . -name .cache`; do
   for f in `grep -v "^[19].*$" $cache | cut -f2 | cut -c3-`; do
--- gofish/gofish-www.conf.orig	2010-10-04 05:03:49.000000000 +0200
+++ gofish/gofish-www.conf	2026-04-11 16:19:31.514656660 +0200
@@ -4,10 +4,10 @@
 # editing a line.
 
 # The root directory for the web site(s)
-root = /var/www/html
+root = /home/services/gopherd
 
 # The logfile
-logfile = /var/log/gofish.log
+logfile = /var/log/gofish/gofish.log
 
 # The pidfile (used when starting/stopping the server)
 pidfile = /var/run/gofish.pid
--- gofish/gofish.5.orig	2010-10-04 05:03:49.000000000 +0200
+++ gofish/gofish.5	2026-04-11 16:20:18.914399873 +0200
@@ -67,8 +67,8 @@ and port if necessary.
 .SH EXAMPLE
 .nf
 # GoFish Gopher Server configuration file
-root = /var/gopher
-logfile = /var/log/gopherd
+root = /home/services/hopherd
+logfile = /var/log/gofish/gopherd.log
 pidfile = /var/run/gopherd.pid
 port = 70
 uid = 13
--- gofish/gofish.conf.orig	2010-10-04 05:03:49.000000000 +0200
+++ gofish/gofish.conf	2026-04-11 16:21:55.930540958 +0200
@@ -4,10 +4,10 @@
 # editing a line.
 
 # The root directory for the gopherspace
-;root = /var/gopher
+;root = /home/services/gopherd
 
 # The logfile
-;logfile = /var/log/gopherd.log
+;logfile = /var/log/gofish/gopherd.log
 
 # The pidfile (used when starting/stopping the server)
 ;pidfile = /var/run/gopherd.pid
--- gofish/gofish.h.orig	2010-10-07 08:15:31.000000000 +0200
+++ gofish/gofish.h	2026-04-11 16:23:38.393319203 +0200
@@ -49,7 +49,7 @@
 
 /* If you leave GOPHER_HOST unset, it will default to your
  * your hostname. */
-#define GOPHER_LOGFILE	LOCALSTATEDIR "/log/gofish.log"
+#define GOPHER_LOGFILE	LOCALSTATEDIR "/log/gofish/gopherd.log"
 #define GOPHER_PIDFILE	LOCALSTATEDIR "/run/gofish.pid"
 #define GOPHER_CONFIG	SYSCONFDIR "/gofish.conf"
 /* #define GOPHER_HOST		"" */
diff -u gofish-0.20.orig/gopherd.1 gofish-0.20/gopherd.1
--- gofish-0.20.orig/gopherd.1	Fri Aug 16 19:30:45 2002
+++ gofish-0.20/gopherd.1	Sat Nov  2 14:53:12 2002
@@ -1 +1 @@
-.so man1/gofish.1
+.so gofish.1
